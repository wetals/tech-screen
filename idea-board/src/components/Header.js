import React from "react";

const Header = () => {
  return (
    <header className="header">
      <h2>Get inspired</h2>
      <div className="sortBy">
        <span className="sortBy__label">Sort by</span>
        <select name="sortBy__dropdown">
          <option value="lastUpdated">Last Updated</option>
          <option value="alphabetically">Alphabetically</option>
        </select>
      </div>
    </header>
  );
};

export default Header;
