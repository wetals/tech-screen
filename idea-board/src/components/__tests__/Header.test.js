import React from "react";
import { shallow } from "enzyme";
import renderer from "react-test-renderer";

import Header from "../Header";

test("Header renders a snapshot properly", () => {
  const tree = renderer.create(<Header />).toJSON();
  expect(tree).toMatchSnapshot();
});

test("Header renders properly", () => {
  const wrapper = shallow(<Header />);
  const element = wrapper.find("h2");
  expect(element.length).toBe(1);
  expect(element.text()).toBe("Get inspired");
});
