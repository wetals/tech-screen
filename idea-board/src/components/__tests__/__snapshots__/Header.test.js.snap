// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header renders a snapshot properly 1`] = `
<header
  className="header"
>
  <h2>
    Get inspired
  </h2>
  <div
    className="sortBy"
  >
    <span
      className="sortBy__label"
    >
      Sort by
    </span>
    <select
      name="sortBy__dropdown"
    >
      <option
        value="lastUpdated"
      >
        Last Updated
      </option>
      <option
        value="alphabetically"
      >
        Alphabetically
      </option>
    </select>
  </div>
</header>
`;
